src/components/ExpandedNode/ExpandedNode.tsx

```diff
@@ -130,18 +128,15 @@ function renderBranch(documentation: DocumentationResource | undefined): ReactEl
       }
     }
   } else {
+    const values: string[] = pathwayState.transitions.map(transition => {
+      const description = transition.condition?.description;
+      return description ? description : '';
```

Comment:
Not worth changing by itself but does our babel setup support the null coalescer? `??` , ex `description = transition.condition?.description ?? ''`

Reply:
It should since this is used in `Graph.tsx`